component-template/{{ cookiecutter.slug }}/Makefile

```diff
@@ -0,0 +1,34 @@
+MAKEFLAGS += --warn-undefined-variables
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DEFAULT_GOAL := all
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+JSONNET_FILES   ?= $(shell find . -type f -name '*.*jsonnet' -or -name '*.libsonnet')
+JSONNETFMT_ARGS ?= --in-place
+
+YAML_FILES      ?= $(shell find . -type f -name '*.yaml' -or -name '*.yml')
+YAMLLINT_ARGS   ?= --no-warnings
+YAMLLINT_CONFIG ?= .yamllint.yml
+
+.PHONY: all
+all: lint
+
+.PHONY: lint
+lint: lint_jsonnet lint_yaml
+
+.PHONY: lint_jsonnet
+lint_jsonnet: $(JSONNET_FILES)
+	jsonnetfmt $(JSONNETFMT_ARGS) --test -- $?
+
+.PHONY: lint_yaml
+lint_yaml: $(YAML_FILES)
+	yamllint -f parsable -c $(YAMLLINT_CONFIG) $(YAMLLINT_ARGS) -- $?
```

Comment:
I realised that if we'd use Docker in the Makefile, we could get rid of our custom CI image again and wouldn't require yamllint and jsonnet to be installed locally.
What do you think?